runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegistrationView/UsersRegistrationView.tsx

```diff
@@ -156,11 +160,11 @@ const UsersRegistrationView: FC = () => {
 
     return (
         <>
-            <DeleteUserDialog
+            {hasAdminAccess && <DeleteUserDialog
```

Comment:
again, wy to add this if TENANT_ADMIN can delete users too? (as far as I know he/she should be able to)